includes/class-wc-siftscience-html.php

```diff
@@ -283,6 +283,7 @@ public function display_stats_tables( $stats, $url ) {
 		public function display_debugging_info( $ssl_data, $ssl_url, $log_url, $logs ) {
 			?>
 			<h2>
+				<img src="" alt="Copy to clipboard" />
```

Comment:
can figure out image location 
'images/clipboard.png'

Reply:
You need to use the `plugins_url` function. Example here:

https://github.com/Fermiac/woocommerce-siftscience/blob/0b701fa72946579bca65484b6dbd3d2c92ee2bf2/includes/class-wc-siftscience-orders.php#L64

Reply:
So you might write something like: 

```
plugins_url( 'images/clipboard.png', dirname( __FILE___ ) )
```